.buildkite/pipeline.yaml

```diff
@@ -4,7 +4,7 @@ steps:
   # still saves about 20sec to pin the image to a SHA256 after an upgrade.
   - label: "Build container"
     # Nb: `if` skips the step if the expression evaluates to false
-    if: build.pull_request.repository.fork == false
+    if: build.pull_request.repository.fork != true
```

Comment:
Wait... WHAT?!

Reply:
Please consider donating to "ASSOCIATION OF COMPUTER SCIENTISTS AND PRACTICIONERS FOR THE ABOLISHMENT OF YAML AND BOOLEAN LOGIC" (AOCSAPFTAOYABL)

Reply:
I'll donate some RAD